client-app/modules/customer-reviews/components/product-reviews.vue

```diff
@@ -1,57 +1,138 @@
 <template>
   <VcWidget :title="$t('common.labels.feedback')" prepend-icon="chat" size="lg" class="text-sm">
-    <div class="mb-8 lg:flex lg:justify-between">
-      <div v-if="rating" class="flex">
-        <span class="mr-2 content-center">{{ $t("common.labels.rating") }}:</span>
-        <ProductRating :rating="rating" class="font-bold" />
-      </div>
+    <template v-if="reviews.length">
+      <div class="mb-4 lg:flex lg:justify-between">
+        <div v-if="rating" class="flex">
+          <span class="mr-2 content-center">{{ $t("common.labels.rating") }}:</span>
+          <ProductRating :rating="rating" class="font-bold" />
+        </div>
 
-      <div class="max-lg:mt-3 lg:flex">
-        <span class="mr-2 content-center font-bold max-lg:hidden">
-          {{ $t("common.labels.sort_by_date") }}
-        </span>
-        <VcSelect
-          v-model="sortByDate"
-          :items="sortByDateItems"
-          :disabled="fetching"
-          text-field="label"
-          value-field="id"
-          @change="changeSortByDate"
-        />
+        <div v-if="reviews.length" class="max-lg:mt-3 lg:flex">
+          <span class="mr-2 content-center font-bold max-lg:hidden">
+            {{ $t("common.labels.sort_by_date") }}
+          </span>
+
+          <VcSelect
+            v-model="sortByDate"
+            :items="sortByDateItems"
+            :disabled="fetching"
+            text-field="label"
+            value-field="id"
+            @change="changeSortByDate"
+          />
+        </div>
       </div>
-    </div>
 
-    <div v-for="review in reviews" :key="review.id" class="border-1 my-4 border-b">
-      <div class="my-4 flex justify-between">
-        <div>
-          {{ review.userName }}
-          <div class="text-neutral-400">
-            {{ $d(review.createdDate) }}
+      <div class="divide-y rounded border">
+        <div v-for="review in reviews" :key="review.id" class="space-y-2 p-4">
+          <div class="flex justify-between gap-2">
+            <div>
+              {{ review.userName }}
+              <div class="text-neutral-400">
+                {{ $d(review.createdDate) }}
+              </div>
+            </div>
+
+            <ReviewRating :rating="review.rating" read-only />
+          </div>
+
+          <div>
+            {{ review.review }}
           </div>
         </div>
+      </div>
 
-        <ReviewRating :rating="review.rating" />
+      <div class="mb-6 mt-5 text-center sm:flex sm:flex-wrap sm:items-center sm:gap-3">
+        <VcButton
+          v-if="isAuthenticated && canLeaveFeedback && !!reviews.length && !reviewFormVisible && !reviewSubmitted"
```

Comment:
reviewFormVisible depended from reviews. Here also you use reviews. Difficult to understand what is happening. Can you remove  reviewFormVisible?